addons/web/static/src/search/search_bar/search_bar.scss

```diff
@@ -10,10 +10,22 @@
     }
 
     animation: animate .25s ease-out;
+    max-width: 100%;
 
     .btn, .btn:focus {
         // Boostrap plays around with the z-index of buttons, but this is not needed here
         // It was causing issues with the hotkey overlay
         z-index: 0;
     }
+
+    .o_facet_values {
+        overflow: hidden;
+    }
+
+    .o_facet_value {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
```

Comment:
I attempted using the utility class (text-truncate), but some test cases failed (https://runbot.odoo.com/runbot/build/71784623)

Reply:
I don't see how this error is related to the utility class...

Reply:
The error occurs when calling `dateSearchFacetElement.querySelectorAll('.o_facet_value')` in the (https://github.com/odoo/odoo/blob/18.0/addons/project/static/tests/legacy/burndown_chart_tests.js#L226). This happens because when the document evaluates the XPath expression on the target, the result of `result.singleNodeValue` is `null`. This is due to the presence of the 'text-truncate' class in the `<small>` element, which affects the evaluation.

Reply:
Fair enough. This test needs to be adapted to avoid this kind of issue (the idea of the test is still right, though).

Reply:
I have made the necessary changes to the test case.